docs/source/running-jobs/environment-variables.rst

```diff
@@ -45,7 +45,22 @@ Using in ``setup`` and ``run``
 
 All user-specified environment variables are exported to a task's ``setup`` and ``run`` commands (i.e., accessible when they are being run).
 
-For example, this is useful for passing secrets to the task (see below).
+For example, this is useful for passing secrets (see below) or passing configurations:
+
+.. code-block:: yaml
+
+    # Sets default values for some variables; can be overridden by --env.
+    envs:
+      MODEL_NAME: decapoda-research/llama-65b-hf
+
+    run: |
+      python train.py --model_name ${MODEL_NAME} <other args>
+
+.. code-block:: console
+
+    $ sky launch --env MODEL_NAME=huggyllama/llama-7b task.yaml  # Override.
```

Comment:
should we use the llama model from the same hf repo?

Reply:
Done!